### PR TITLE
`redisenterprise` - fix tests for 4.0

### PR DESCRIPTION
**Pull request by mbfrahry**

### CODE REVIEW

File: internal/services/redisenterprise/redis_enterprise_database_resource.go [hunk 1]
```diff
@@ -341,7 +341,6 @@ func resourceRedisEnterpriseDatabaseRead(d *pluginsdk.ResourceData, meta interfa
 	}
 
 	d.Set("name", id.DatabaseName)
-	d.Set("resource_group_name", id.ResourceGroupName)
```
Comment (jackofallops) [inline internal/services/redisenterprise/redis_enterprise_database_resource.go:344]: This probably needs to be wrapped in the beta flag for now and removed post-4.0 instead?

**Review by jackofallops — CHANGES_REQUESTED** [review]
Thanks @mbfrahry - One comment to take a look at.

Thanks!

File: internal/services/redisenterprise/redis_enterprise_database_resource.go (hunk 1 above)
Comment (mbfrahry) [reply internal/services/redisenterprise/redis_enterprise_database_resource.go:344]: Ooops! Missed this one. It's actually wrapped in that beta flag check just below this.

**Review by stephybun — APPROVED** [review]
Thanks @mbfrahry LGTM 👍

**Pull request merged**